conf/messages.en

```diff
@@ -256,8 +256,11 @@ assessment.hint = Select one answer.
 assessment.exemption = {0} - {1}
 assessment.exemption.none = None of the above
 
-ukimsKickOut.title = ukimsKickOut
-ukimsKickOut.heading = ukimsKickOut
+ukimsKickOut.title = UKIMS kick out page
+ukimsKickOut.h1 = You need to apply for the UK Internal Market Scheme (UKIMS)
+ukimsKickOut.p1 = You cannot use Trader Goods Profile (TGP) if you are not UKIMS registered.
+ukimsKickOut.p2 = Find out how to
+ukimsKickOut.linkText = register for UKIMS
```

Comment:
linkText is missing (opens in a new tab).

Reply:
I thought that was just giving an instruction to the dev that it just needs to open in a new tab >_< will clarify with design lol

Reply:
checked and changed! :)